wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession+UserSession.swift

```diff
@@ -294,6 +294,29 @@ extension ZMUserSession: UserSession {
         let featureRepository = FeatureRepository(context: syncContext)
         return GetMLSFeatureUseCase(featureRepository: featureRepository)
     }
+
+    @MainActor
```

Comment:
Why `@MainActor`? I think it should be removed.

Reply:
It crashes otherwise